packages/create-youtrack-app/_templates/init/vite-app/@types/globals.d.ts.t

```diff
@@ -16,12 +16,10 @@ export interface HubService {
 }
 
 interface BaseAPILayer {
-  alert: (...args: Parameters<(typeof AlertService)['addAlert']>) => void;
+  alert: (message: string, type?: AlertType | undefined, timeout?: number) => void;
   enterModalMode: Promise<() => void>;
   exitModalMode: Promise<() => void>;
-  /** @deprecated use "closeWidget()" method instead */
```

Comment:
Could you please explain why was this change made?

Reply:
I mixed branches up a bit, copied content of this file from another branch. Thanks, will fix